### PR TITLE
Added fancy pde:// install button

### PR DESCRIPTION
**Pull request by SableRaf**
Added an install button point at the `.pdex` file using the custom `pde://` uri protocol.

The button includes a tooltip with fallback instructions.

image

### Context
Processing 4.2 added the new `pde://` protocol which allows you to open Processing sketches and contributions in the PDE directly from the browser. It builds up on the previous addition of bundled sketches and contributions with the `.pdez` and `.pdex` formats.

### CODE REVIEW

**Comment by SableRaf** [comment]
Hmmm... Looks like the changes I made are causing Processing to see the source as untrusted. ~~This didn't happen with a simple link as it is currently set up on [the production deploy](https://android.processing.org/install)~~. Nevermind it's also broken today. Pretty sure it worked when I tried it yesterday.

cc @mingness @stefterv 

image

**Comment by Stefterv** [comment]
Looking at the source code it will always show that dialog: https://github.com/benfry/processing4/blob/6a2cf8cda35552c62a1a794bb1e20f43fe8ffcda/app/src/processing/app/Base.java#L1245

**Comment by SableRaf** [comment]
Oh my bad! You're right. This is a necessary warning. I just got confused by the `scheme8019...` title which really obfuscates the question mark. I wonder where that string comes from 🤔 It should be the name of the contribution. 

Edit: Getting closer...

```Java
String baseName = contribFile.getName();
baseName = baseName.substring(0, baseName.length() - CONTRIB_BUNDLE_EXT.length());
```

File: es/gallery/index.html
Comment (Stefterv) [inline es/gallery/index.html:1]: Can these side-wide changes be part of a different PR? It hides what you actually did

Comment (SableRaf) [reply es/gallery/index.html:1]: On it! I'm splitting this PR into several more specific ones.

**Comment by SableRaf** [comment]
Closing and splitting this PR into three: https://github.com/processing/processing-android-website/pull/30 https://github.com/processing/processing-android-website/pull/29 https://github.com/processing/processing-android-website/pull/28
